ground/build.gradle

```diff
@@ -86,6 +86,7 @@ android {
         buildConfigField "String", "EMULATOR_HOST", "\"10.0.2.2\""
         buildConfigField "int", "FIRESTORE_EMULATOR_PORT", "8080"
         buildConfigField "int", "AUTH_EMULATOR_PORT", "9099"
+        buildConfigField "String", "SIGNUP_FORM_LINK", "\"\""
```

Comment:
I think that's a fair point. We should keep all modifiable configs at a single location. 

However, `Config.kt` is supposed to be application level configuration and shouldn't be modified by forks.
Any param defined in `build.gradle` can be provided from command line while generating the apk (like  `gradlew assembleRelease -PparamName=paramValue`)

Reply:
The offline imagery stuff should be configured by the config doc in Firestore, I think having it in `Config` is a temporary solution.

It sounds like build.gradle is more flexible, since it doesn't require hard-coded changes for different builds, so if we have to choose one, it sounds like the right place.